linked-list/src/example/java/Element.java

```diff
@@ -0,0 +1,52 @@
+public final class Element {
```

Comment:
This class should be package private. Or just be a private static inner class of `Deque`.